Synview-frontend/src/components/MentorInviteForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { useInviteMentorMutation } from "../services/apiSlice.ts";
+import { useAppSelector } from "../hooks.ts";
+import { ProjectRoles } from "../../../common/schemas.ts";
+export default function MentorInviteForm() {
+  const [invitedMentorId, setInvitedMentorId] = useState("");
+
+  const project_id = useAppSelector(
+    (state) => state.inviteMentorModal.project_id
+  );
+  const user_id = useAppSelector((state) => state.inviteMentorModal.user_id);
+  const [inviteMentor] = useInviteMentorMutation();
+  const handleInviteForm = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      if (project_id && user_id) {
+        await inviteMentor({
+          role: ProjectRoles.REVIEWER,
+          invited_user_id: parseInt(invitedMentorId),
+          inviting_user_id: user_id,
+          invited_project_id: project_id,
+        });
+        setInvitedMentorId("");
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return (
+    <div>
+      <fieldset className="fieldset border-1 p-4 rounded-box border-neutral-600 bg-neutral-800">
+        <div className="flex flex-col justify-center">
+          <form onSubmit={handleInviteForm}>
+            <label>Id</label>
+            <input
+              className="input bg-neutral-700 w-full"
+              typeof="text"
+              placeholder="id"
+              id="Username"
+              value={invitedMentorId}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                setInvitedMentorId(e.target.value);
+              }}
+            ></input>
```

Comment:
```suggestion
            />
```
Nit: `input` is one of the classic elements you self close. But with React you can do it with nearly every element/component.